themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
See my comment above about improving readability. Are there any other outstanding issues from this conversation thread that we need to worry about?